pkg/reconciler/broker/broker_test.go

```diff
@@ -852,6 +852,38 @@ func channelCRD() metav1.TypeMeta {
 	}
 }
 
+func livenessProbe() *corev1.Probe {
+	return &corev1.Probe{
+		Handler:             corev1.Handler{
+			HTTPGet: &corev1.HTTPGetAction{
+				Path:        "/healthz",
+				Port:        intstr.IntOrString{Type: intstr.Int, IntVal: 8080},
+			},
+		},
+		InitialDelaySeconds: 5,
+		PeriodSeconds:       2,
+	}
+}
+
+func readinessProbe(containerName string) *corev1.Probe{
```

Comment:
Is the intention to only add the probe if it is the filter container? If so, why ever call this function for the ingress container?